action.yml

```
@@ -32,6 +32,10 @@ inputs:
     description: Exclude unsafe cops.
     required: false
     default: "true"
+  working-directory:
```

Can you use snake_case for this input identifier? 

```suggestion
  working_directory:
```

It is true that GitHub's workflow syntax uses chain-case like `working-directory:` for key names, but it is not a good idea to mix them like `only_safe` and `working-directory` here.

---

I agree with you. I'll accept that.